feat(HTTP): Refresh request method pages (#35745)
mdn/content: files/en-us/web/http/methods/head/index.md

{{HTTPSidebar}}

> The **HTTP `HEAD` request method** requests the metadata of a resource in the form of [headers](/en-US/docs/Web/HTTP/Headers) that the server would have sent if the {{HTTPMethod("GET")}} method was used instead.

Reviewer: ```suggestion
The **`HEAD`** HTTP method requests the metadata of a resource in the form of [headers](/en-US/docs/Web/HTTP/Headers) that the server would have sent if the {{HTTPMethod("GET")}} method was used instead.
```

Reviewer: OR maybe

```suggestion
The **`HEAD`** HTTP method requests the metadata of a resource.
This results in a response that includes the [headers](/en-US/docs/Web/HTTP/Headers) that the server would have sent for the equivalent {{HTTPMethod("GET")}} method, and no message body.
```

{{HTTPSidebar}}

> The **`HEAD`** HTTP methos requests the metadata of a resource in the form of [headers](/en-US/docs/Web/HTTP/Headers) that the server would have sent if the {{HTTPMethod("GET")}} method was used instead.

Reviewer: ```suggestion
The **`HEAD`** HTTP method requests the metadata of a resource in the form of [headers](/en-US/docs/Web/HTTP/Headers) that the server would have sent if the {{HTTPMethod("GET")}} method was used instead.
```

## Syntax

```http
HEAD <request-target>["?"<query>] HTTP/1.1
```

- `<request-target>`
  - : Identifies the target resource of the request when combined with the information provided in the {{HTTPHeader("Host")}} header.
    This is an absolute path (e.g., `/path/to/file.html`) in requests to an origin server, and an absolute URL in requests to proxies (e.g., `http://www.example.com/path/to/file.html`).
- `<query>` {{optional_inline}}
  - : An optional query component preceded by a question-mark `?`.
    Often used to carry identifying information in the form of `key=value` pairs.

## Examples

### Successfully retrieving resource metadata

> The following curl command creates a `HEAD` request for `example.com`:

Reviewer 2: ```suggestion
The following `curl` command creates a `HEAD` request for `example.com`:
```

HEAD /index.html
```

## Examples

### Successfully retrieving resource metadata

The following curl command creates a `HEAD` request for `example.com`:

```bash
curl --head example.com
```

> This is the equivalent to a `GET` request, except the server shouldn't send any content in the response.

Reviewer: Maybe
```suggestion
This is the equivalent to a `GET` request, except the server shouldn't include a message body in the response.
```

It creates an HTTP request that looks like this:

```http
HEAD / HTTP/1.1
Host: example.com
User-Agent: curl/8.6.0
Accept: */*
```

The server sends back a {{HTTPStatus("200", "200 OK")}} response comprised only of headers.

> The response is effectively metadata that describes the resource instead of the resource itself (some [caching](/en-US/docs/Web/HTTP/Caching) headers are omitted for brevity):

Reviewer: ```suggestion
The response is effectively metadata that describes the resource instead of the resource itself (some [caching](/en-US/docs/Web/HTTP/Caching) headers are omitted in this example for brevity):
```
